adapters/rest-adapter-generator/src/test/java/uk/gov/justice/services/adapters/rest/generator/DefaultGenerator_ResourceCodeStructureTest.java

```diff
@@ -67,6 +70,7 @@ public void shouldGenerateAnnotatedResourceInterface() throws Exception {
                 ).build(),
                 configurationWithBasePackage(BASE_PACKAGE));
 
+
         Class<?> interfaceClass = compiler.compiledInterfaceOf(BASE_PACKAGE);
 
         assertThat(interfaceClass.isInterface(), is(true));
```

Comment:
I think this is mainly a reordering of imports, but its difficult to see what has actually changed.